core/test/fixtures/fraggle-rock/artifacts/step0/options.json

```diff
@@ -1,5 +1,9 @@
 {
   "flags": {
-    "skipAboutBlank": true
+    "skipAboutBlank": true,
+    "skipAudits": [
+      "uses-http2",
```

Comment:
why is `uses-http2` skipped? #13715 doesn't seem to say

Reply:
It was skipped in the config. If we're automatically skipping an audit it should be done in an append fashion.

See changes in `user-flow.js` for how this is done.

Reply:
right, I mean why is skipping `uses-http2` in the config? :)

https://github.com/GoogleChrome/lighthouse/blob/80709e6c7dd126af3af34d75985a79fbcf7f429e/core/scripts/update-flow-fixtures.js#L74-L80

Reply:
Because I thought testing a non-default config would be useful.